catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/handlers/CqlTransformHandler.java

```diff
@@ -217,4 +231,46 @@ private void attachFileToResponse(
         "Successfully output file using transformer id {}",
         queryResponseTransformer.getProperty("id"));
   }
+
+  private Map<String, Serializable> cswTransformArgumentsAdapter() {
+    Map<String, Serializable> args = new HashMap<>();
+    args.put(CswConstants.IS_BY_ID_QUERY, true);
+    return args;
+  }
+
+  private Map<String, Serializable> csvTransformArgumentsAdapter(
+      Map<String, Serializable> arguments) {
+    String columnOrder = "\"columnOrder\":" + (String) mapper.toJson(arguments.get("columnOrder"));
+    String hiddenFields =
+        "\"hiddenFields\":" + (String) mapper.toJson(arguments.get("hiddenFields"));
+    String columnAliasMap =
+        "\"columnAliasMap\":" + (String) mapper.toJson(arguments.get("columnAliasMap"));
+    String csvBody = "{" + columnOrder + "," + columnAliasMap + "," + hiddenFields + "}";
+
+    CsvTransform queryTransform = mapper.readValue(csvBody, CsvTransform.class);
+
+    Set<String> hiddenFieldsSet =
+        queryTransform.getHiddenFields() != null
+            ? queryTransform.getHiddenFields()
+            : Collections.emptySet();
+
+    List<String> columnOrderList =
+        queryTransform.getColumnOrder() != null
+            ? queryTransform.getColumnOrder()
+            : Collections.emptyList();
+
+    Map<String, String> aliasMap =
+        queryTransform.getColumnAliasMap() != null
+            ? queryTransform.getColumnAliasMap()
+            : Collections.emptyMap();
+
+    Map<String, Serializable> args =
+        ImmutableMap.<String, Serializable>builder()
+            .put("hiddenFields", new HashSet<>(hiddenFieldsSet))
```

Comment:
do we have to make new collections for these?  `.put("hiddenFields", hiddenFieldsSet)`